grpc/proto/space.proto

```diff
@@ -260,6 +301,9 @@ message ListDirectoryEntry {
   string updated = 6;
   string fileExtension = 7;
   string ipfsHash = 8;
+  bool local = 9;
```

Comment:
Maybe I'd call this `isLocallyAvailable` or something like that to be more semantic.

Reply:
Fixed in [35ad3a0](https://github.com/FleekHQ/space-daemon/pull/109/commits/35ad3a05949cc481001b673672990921b33a4951).